scrapy_autounit/utils.py

```diff
@@ -101,10 +102,15 @@ def add_sample(index, test_dir, test_name, data):
     fixture_name = 'fixture%s' % str(index)
     filename = fixture_name + '.bin'
     path = test_dir / filename
-    data = compress_data(pickle_data(data))
+    info = pickle_data({
+        'data': pickle_data(data),
+        'encoding': encoding,
+        'fixture_version': FIXTURE_VERSION,
```

Comment:
So if the fixture version is in the pickled data, if you decide to switch away from pickle in the future you won't be able to tell that from the version because to get the version you need to unpickle first.

One option is to say reserve the first 4 bytes of the file for the version number, which would give you 64k version numbers (or 2 bytes might be fine :sweat_smile:)

Reply:
Ah, I see you're pickling and then pickling again, so the inner format could change.  Ignore me, I think this is a good solution.